web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/svp/BekreftSvangerskapspengerOppdaterer.java

```diff
@@ -336,12 +336,14 @@ private SvpTilretteleggingEntitet mapNyTilrettelegging(SvpArbeidsforholdDto arbe
         }
 
         if (arbeidsforholdDto.getAvklarteOppholdPerioder() != null) {
-            arbeidsforholdDto.getAvklarteOppholdPerioder().forEach( avklartOpphold -> {
-                var nyttOpphold = SvpAvklartOpphold.Builder.nytt()
-                    .medOppholdPeriode(avklartOpphold.fom(), avklartOpphold.tom())
-                    .medOppholdÅrsak(avklartOpphold.oppholdÅrsak())
-                    .build();
-                nyTilretteleggingEntitetBuilder.medAvklartOpphold(nyttOpphold);
+            arbeidsforholdDto.getAvklarteOppholdPerioder().stream()
+                .filter(oppholdDto -> !oppholdDto.forVisning()) //Vi viser opphold fra IM - disse skal ikke lagres i grunnlaget
```

Comment:
Har du hørt med Tor om han liker dette med "forVisning". Du har flere mulige strategier for ferie
* Vise ferie fra IM, ikke tillate endring - men be SBH be om ny IM
* Vise ferie fra IM, tillate å "endre" / legge til ferie - da kommer ferie fra både IM og TLR-grunnlag og må flettes - som kan bli litt kaos i oppførsel
* Vise ferie fra IM, bekrefte i TLR - "hvis ferie i gjeldendeTLR - ikke ta med ferie fra IM" til gui eller regler. Hvis ikke ferie i gjeldendeTLR - ta med fra ferie IM - men lagre det som SBH bekrefter